tests/kernel/before_nfs_test.pm

```diff
@@ -11,13 +11,22 @@ use testapi;
 use serial_terminal "select_serial_terminal";
 use utils;
 use lockapi;
+use power_action_utils 'power_action';
 
 sub run {
     my ($self) = @_;
 
     select_serial_terminal;
-    systemctl 'stop ' . $self->firewall;
+    systemctl '--now disable ' . $self->firewall;
     set_hostname(get_var("HOSTNAME", "susetest"));
+
+    power_action('reboot');
+    $self->wait_boot;
+    select_serial_terminal;
+
+    assert_script_run('ip addr');
```

Comment:
it seems you print just info here. i suggest to just replace `assert_script_run` with `script_run`

Reply:
I will change this in a future NFS PR, I have removed it for now.